db/migrate/20211101080623_add_index_on_plage_ouverture_first_day.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddIndexOnPlageOuvertureFirstDay < ActiveRecord::Migration[6.0]
+  def change
+    add_index :plage_ouvertures, :first_day
+    add_index :plage_ouvertures, :recurrence_until
+    add_index :plage_ouvertures, :recurrence, where: "recurrence IS NOT NULL"
```

Comment:
Mmmmmmmmmm l’index sur `:first_day` et sur `recurrence IS NOT NULL` ont l’air d’être utilisés dans le `scope :in`, mais pour l’index sur `recurrence_until`, j’ai l’impression que ce qu’on veut c’est un index sur le range, non? On en a fait un pour `RdvAddRangeIndex`, tu vois comment vérifier que l’index est utilisé avec `.explain`?

Reply:
je ne suis pas sur non.